scripts/get-courses.py

```diff
@@ -65,67 +89,40 @@ def make_request(endpoint):
                 logging.error(f"results not in JSON object")
                 raise RuntimeError(f"results not in JSON object")
             if 'next' not in response_json.keys() or response_json['next'] is None:
-                next_available = False                
+                next_available = False
             courses_json += response_json['results']
         except Exception as e:
             logging.error(f"Error while parsing JSON object of assignatures/?page={page_num}: {e}")
             raise RuntimeError(f"Error while parsing JSON object of assignatures/?page={page_num}: {e}")
     else:
         logging.error(f"API call to assignatures/?page={page_num} failed: <{response.status_code}>: {response.text}")
         raise RuntimeError(f"API call to assignatures/?page={page_num} failed: <{response.status_code}>: {response.text}")
-    
+
     page_num += 1
 
 df = pd.DataFrame(courses_json)
 df = df[['id', 'nom', 'credits', 'vigent']]
 df = df[df['vigent'] == 'S']
 df = df.drop(columns=['vigent'])
-df = df.rename(columns={'id': '_id', 'nom': 'name', 'credits': 'creditHours'})
+df = df.rename(columns={'id': 'slug', 'nom': 'name', 'credits': 'creditHours'})
```

Comment:
It recommended to use an `ObjectId()` as _id to improve performance in the queries